### PR TITLE
Bug 1947794: Update operatorhub config object to v1

### PR DESCRIPTION
**Pull request by kevinrizza**
Update the operatorhub config API to CRD v1

### CODE REVIEW

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

File: config/v1/0000_03_config-operator_01_operatorhub.crd.yaml [hunk 1]
```diff
@@ -14,92 +14,93 @@ spec:
     plural: operatorhubs
     singular: operatorhub
   scope: Cluster
-  preserveUnknownFields: false
   subresources:
```
Comment (soltysh) [inline config/v1/0000_03_config-operator_01_operatorhub.crd.yaml:17]: This is misplaced, should be under versions.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by kevinrizza** [comment]
@soltysh Updated and verify is passing now. Ptal :)

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**